rainforest-cli.go

```diff
@@ -206,9 +206,13 @@ func main() {
 					Usage: "Run your tests using specified `ENVIRONMENT`. Otherwise it will use your default one.",
 				},
 				cli.StringFlag{
-					Name: "crowd",
-					Usage: "Run your tests using specified `CROWD`. Available choices are: default, automation, automation_and_crowd " +
-						"or on_premise_crowd. Contact your CSM for more details.",
+					Name: "execution-method",
+					Usage: "Run your tests using specified `EXECUTION_METHOD`. Available choices are: crowd, automation, automation_and_crowd " +
+						"or on_premise. Some methods may not be available for your account.",
```

Comment:
```suggestion
						"and on_premise. Some methods may not be available for your account.",
```
I think this sounds more natural because `on_premise` is _also_ part of the choices.

Should we add this here too as in the README:  "For more information, contact us at help@rainforestqa.com"